### PR TITLE
Fix AnnotatedImage styling + add demo

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

Closes: #4607

I think it was a css issue. 

annotated_image_resize

To test, run the image_segmentation demo or the demo added in this PR.

Also noticed that AnnotatedImage has no associated component demo so adding one.

image

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests & Changelog

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

1. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`
  
1. Unless the pull request is labeled with the "no-changelog-update" label by a maintainer of the repo, all pull requests must update the changelog located in `CHANGELOG.md`:

Please add a brief summary of the change to the Upcoming Release section of the `CHANGELOG.md` file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

### CODE REVIEW

File: js/app/src/components/AnnotatedImage/AnnotatedImage.svelte [hunk 1]
```diff
@@ -125,6 +125,11 @@
 </Block>
 
 <style>
+	.base-image {
```
Comment (freddyaboulton) [inline js/app/src/components/AnnotatedImage/AnnotatedImage.svelte:128]: The base image is given this class previously but it was never defined anywhere

**Comment by pngwn** [comment]
Thanks @freddyaboulton will take a look at this very shortly!

**Comment by freddyaboulton** [comment]
@hannahblair not seeing that behavior when I built locally. Tried dragging horizontally and diagonally 🤔 

annotated_image_resize_3
annotated_image_resize_2

**Comment by freddyaboulton** [comment]
@hannahblair I think it's doing the right thing? I actually didn't know you could test the mobile view!! Very cool - thank you

annotated_image_resize_mobile

**Review by pngwn — APPROVED** [review]
Tested + looks good. Thanks @freddyaboulton!

**Comment by freddyaboulton** [comment]
Thanks so much for the reviews all! Already made a note to add a visual test for this in the future.

**Pull request merged**